torch/_dynamo/symbolic_convert.py

```diff
@@ -1782,7 +1782,7 @@ def inline_call_(parent, func, args, kwargs):
             func.get_filename()
         ) and not skipfiles.is_torch_inline_allowed(func.get_filename()):
             unimplemented(
-                f"inline in skipfiles: {func.get_name()} {func.get_filename()}"
+                f"inline in skipfiles: {func} | {func.get_name()} {func.get_filename()}"
```

Comment:
I'm not sure what the point of printing func here is as well? Is `func.get_name()` not giving you useful information?

Reply:
It is not, I'll add a before after example :)

Reply:
```
inline in skipfiles: _fn /scratch/voz/work/pytorch/torch/_dynamo/eval_frame.py
```
to
```
inline in skipfiles: SGD._init_group | _fn /scratch/voz/work/pytorch/torch/_dynamo/eval_frame.py
```

Reply:
Why is the filename `_dynamo/eval_frame.py` for `SGD._init_group`?  This looks like a bug.

Reply:
I bet its a bug because of indirection, same reason the name is `_fn` - there's a `def _fn` in eval_frame. Lemme look into it...